### ISSUE TITLE
work with "separate groups"

### ISSUE THREAD

**Issue by krueuw** [frankkoch/moodle-mod_studentquiz#298]
We use student quizzes in a course that contains several years of a training occupation. It would be good if it were possible to work with "separate groups" or to use the existing display filtering to only see the currently active actors. This also applies analogously to the high score.

**Comment by dionysius** [comment]
Current active actors are all people enrolled to the course.

Depending if your "groups" are different roles, you can exclude roles in the ranking already in the studentquiz settings:
![image](https://user-images.githubusercontent.com/1341084/101896682-9e859b00-3ba9-11eb-85ff-5efff7e1bf5a.png)

**Comment by krueuw** [comment]
Yeah ok, that's clear, but not what is needed. Under Moodle it is normal to work with separate groups within the courses and still use common activities. E.g. In the general education schools the parallel classes of a year concern, in the vocational school the apprenticeship years of a training occupation, i.e. all in the role of student.
